src/it/scala/nl/wbaa/testkit/AwaitAtMostTrait.scala

```diff
@@ -0,0 +1,18 @@
+package nl.wbaa.testkit
+
+import scala.concurrent.duration._
+import scala.util.Try
+
+
+trait AwaitAtMostTrait {
+  private val maxWaitTimeForContainers: Int = 10 * 60
+
+  lazy val waitAtMostDuration: FiniteDuration = {
+    val secondsTimeout = sys.env.get("INTEGRATION_TEST_TIMEOUT") match {
```

Comment:
The name of the environment variable should indicate the units. So something like: `INTEGRATION_TEST_TIMEOUT_SECONDS`